### PR TITLE
global_4deg setup & docs update - include ITF

### PR DESCRIPTION
**Pull request by nutrik**

### CODE REVIEW

File: veros/setup/global_4deg/global_four_degree.py [hunk 1]
```diff
@@ -119,7 +122,11 @@ def set_coriolis(self, vs):
     def set_topography(self, vs):
         bathymetry_data = self._read_forcing(vs, 'bathymetry')
         salt_data = self._read_forcing(vs, 'salinity')[:, :, ::-1]
-        mask_salt = salt_data == 0.
+        depth = np.array([-4855., -4165., -3575., -2985., -2495., -2005., -1615., -1225.,
```
Comment (dionhaefner) [inline veros/setup/global_4deg/global_four_degree.py:125]: What is this? Is this `vs.zt`?

File: veros/setup/global_4deg/global_four_degree.py [hunk 2]
```diff
@@ -21,10 +21,13 @@ class GlobalFourDegreeSetup(VerosSetup):
     This setup demonstrates:
      - setting up a realistic model
      - reading input data from external files
+     - including Indonesian throughflow
      - implementing surface forcings
      - applying a simple ice mask
 
     `Adapted from pyOM2 <https://wiki.cen.uni-hamburg.de/ifm/TO/pyOM2/4x4%20global%20model>`_.
+
+    Courtesy of Franka Jesse, Utrecht University
```
Comment (dionhaefner) [inline veros/setup/global_4deg/global_four_degree.py:30]: Maybe put some changelog in here or so? It's not like she wrote the whole thing.

File: veros/setup/global_4deg/global_four_degree.py (hunk 1 above)
Comment (nutrik) [reply veros/setup/global_4deg/global_four_degree.py:125]: Dammit! It is. I will change it

File: veros/setup/global_4deg/global_four_degree.py (hunk 2 above)
Comment (nutrik) [reply veros/setup/global_4deg/global_four_degree.py:30]: Good idea!

File: veros/setup/global_4deg/global_four_degree.py [hunk 3]
```diff
@@ -122,10 +124,8 @@ def set_coriolis(self, vs):
     def set_topography(self, vs):
         bathymetry_data = self._read_forcing(vs, 'bathymetry')
         salt_data = self._read_forcing(vs, 'salinity')[:, :, ::-1]
-        depth = np.array([-4855., -4165., -3575., -2985., -2495., -2005., -1615., -1225.,
-                          -935.,  -645.,  -455.,  -265.,  -175.,   -65.,   -35.])
         _salt = salt_data
```
Comment (dionhaefner) [inline veros/setup/global_4deg/global_four_degree.py:127]: This is redundant, just use `salt_data`

Comment (nutrik) [reply veros/setup/global_4deg/global_four_degree.py]: fixed

File: veros/setup/global_4deg/global_four_degree.py [hunk 4]
```diff
@@ -119,6 +124,7 @@ def set_coriolis(self, vs):
     def set_topography(self, vs):
         bathymetry_data = self._read_forcing(vs, 'bathymetry')
         salt_data = self._read_forcing(vs, 'salinity')[:, :, ::-1]
+        salt_data[vs.zt[np.newaxis, np.newaxis, :] <= bathymetry_data[..., np.newaxis]] = 0.
```
Comment (dionhaefner) [inline veros/setup/global_4deg/global_four_degree.py:127]: Let's absorb this into `mask_bathy`:

```python
mask_bathy = vs.zt[np.newaxis, np.newaxis, :] <= bathymetry_data[..., np.newaxis]
```

Makes is a bit clearer what's going on.

**Comment by dionhaefner** [comment]
Damn, I missed that the shapes don't match if you do it this way. Let's revert the last change and we're good to go.

By the way, do you have a new preview image for the gallery?

**Comment by nutrik** [comment]
Alright ;) 
Yes, I produced a new PSI image based on 10 years integration.

**Pull request merged**